regulations/static/regulations/js/source/views/comment-review-view.js

```diff
@@ -49,7 +49,7 @@ var CommentReviewView = Backbone.View.extend({
 
   serialize: function() {
     return {
-      docNumber: this.docNumber,
+      doc_number: this.docNumber,
```

Comment:
Regs.gov doesn't actually use `documentId` when submitting (they use `comment_on`), so let's do this:

1) call this field `fr_id` or something similar
2) ignore it and use a setting in `regulations.tasks.submit_comment`

I think the frontend will have the `comment_on` id some point soon, but there's no point delaying this.